addons/l10n_it_edi/models/account_edi_format.py

```diff
@@ -748,6 +761,12 @@ def _import_fattura_pa_line(self, element, invoice_line_form, extra_info=None):
                     self.env['account.move']._compose_info_message(element, '.'),
                 ))
 
+        # If no taxes were found, try to find fitting taxes
+        if not invoice_line_form.tax_ids:
+            fitting_taxes = invoice_line_form._predict_taxes()
+            if fitting_taxes:
```

Comment:
Maybe more a question for @lordkrandel 

Isn't it possible that we want to detect that there is no taxes in the IT l10n?

Reply:
Didn't understand the question, sorry, can you rephrase?